src/main/java/edu/wright/airviewer2/AIRViewerController.java

```diff
@@ -342,5 +354,65 @@ public void initialize(URL url, ResourceBundle rb) {
         });
 
     }
+    
+    @FXML private TextField textFieldValue;
+    
+    @SuppressWarnings("deprecation")
+   @FXML private void download() throws IOException {
+    System.out.println("textfieldvalue"+textFieldValue.getText());
+    Splitter splitter = new Splitter();
+   // FileChooser fileChooser = new FileChooser();
+   // FileChooser.ExtensionFilter extFilter = new FileChooser.ExtensionFilter("PDF files (*.pdf)", "*.pdf");
+   // fileChooser.getExtensionFilters().add(extFilter);
+   // 
+   // File file = fileChooser.showSaveDialog((Stage) pagination.getScene().getWindow());
```

Comment:
please remove the unwanted cocmmented code

Reply:
Removed the commented code. Please review again.